trigger/task.go

```diff
@@ -774,15 +774,15 @@ func (t *taskTriggers) shouldIncludeTest(sub *event.Subscription, previousTask *
 		return true, nil
 	}
 
-	oldTestResult, ok := t.oldTestResults[test.TestFile]
+	oldTestResult, ok := t.oldTestResults[test.GetDisplayTestName()]
 	// a new test in an existing task is defined as a regression
 	if !ok {
 		return true, nil
 	}
 
 	if isTestStatusRegression(oldTestResult.Status, test.Status) {
 		// try to find a stepback alert
-		alertForStepback, err := alertrecord.FindByTaskRegressionTestAndOrderNumber(sub.ID, test.TestFile, currentTask.DisplayName, currentTask.BuildVariant, currentTask.Project, previousTask.RevisionOrderNumber)
+		alertForStepback, err := alertrecord.FindByTaskRegressionTestAndOrderNumber(sub.ID, test.GetDisplayTestName(), currentTask.DisplayName, currentTask.BuildVariant, currentTask.Project, previousTask.RevisionOrderNumber)
```

Comment:
Is using `GetDisplayTestName` instead of `TestFile` backwards compatible with non-Cedar test results?

Reply:
Yes! If the display name DNE, `GetDisplayTestName` returns `TestFile`.